### DISCUSSION TITLE
Any specific OpenSuSE Build Service repository for Ubuntu 23.04

### DISCUSSION THREAD

**Discussion by pejiji** [abraunegg/onedrive#2399] [Q&A] [answered]
I just moved from Ubuntu 22.10 to Ubuntu 23.04 version.
I had installed the onedrive package following the "ubuntu 22.10" specific instructions.
The package seems to continue to work correctly over Ubuntu 23.04, but I was wondering when there will be a specific openSuSE repository for Ubuntu 23.04, and how to migrate to it when available.

**Comment by abraunegg** [comment, marked as answer]
@pejiji 
There are no immediate plans for an OpenSuSE 23.04 package for the following reasons:

1. Ubuntu 23.04 is a transient release - ie - a non LTS release
2. Ubuntu 23.04 already contains as a default package v2.4.23 which is the latest release
3. All development effort has shifted to v2.5.x series, and it is highly unlikely that there will be a v2.4.24 release - thus that Ubuntu version will be the latest and current for at least the next 6 months

Your choices here are:
 * Use the v2.4.23 package that works for you, but my suggestion is to use a Ubuntu LTS release, not a transient release
 * If keeping this package current + keeping your Operating System current is desirable, potentially think about using something other than Ubuntu.
